### PR TITLE
Week 5 Capstone

### PR DESCRIPTION
**Pull request by mmatongo**
## Project requirements successfully fulfilled
- Used correct GitHub flow.
- Fixed linter errors for HTML, CSS and JS.
- Use GitHub Pages to deploy web pages.
- Use JavaScript to manipulate DOM elements.
- Use objects to store and access data.
- Built the website for both mobile (<768px) and desktop (>768px).
- The home page should have a link in the menu to the about page.
- The logo in the header links to the home page.
- Mobile menu
  - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
The mobile menu should have a close (X) button that closes the menu.
- Dynamic page
  - The section "Featured speakers" should be created dynamically in JavaScript.
  - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

## Home Page

screenshot

screenshot

## About Page

screenshot

## Live Demo
[Live Demo](https://mmatongo.github.io/microverse-capstone/)

## Walkthrough
[Click Here](https://www.loom.com/share/c7c914be69004689994da70535706a25)

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        </div>
+        <br>
+        <div class="conference-image-parent">
+            <div class="conference-image-child">
+                <img src="../bg/warsaw.png" alt="warsaw" class="warsaw">
+            </div>
+            <div class="conference-image-child">
+                <img src="../bg/baires.png" alt="beunos aires" class="baires">
+            </div>
+        </div>
```
Comment (mimipeshy) [inline about/index.html:93]: - [ ] These two images should have a hover color in the mobile version just like you implemented in the desktop version
image

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @mmatongo  🔥 

Even though it will be amazing to see you personalize your capstone, you worked hard in all the instructions that the previous Code Reviewer gives you, your capstone is complete and you are ready for JavaScript, good job👍🏽 

nice

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mmatongo** [comment]
> Now you can merge this branch 🚀


Thank you

File: js/speakers.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+                      <div>
+                        <h3>${moreSpeaker[3].name}</h3>
+                        <p class="desc">${moreSpeaker[3].desc}</p>
+                        <hr class="speakers-line">
+                        <p class="speakerdesc">${moreSpeaker[3].spdesc}</p>
+                      </div>
+                    </article>`;
+
+  return htmlText;
+}
```
Comment (rpire) [inline js/speakers.js:111]: Although your code is working as required and the *Speakers* are being dynamically generated, the code could be visually and conveniently improved by implementing a `for` loop and indexing into this section.

You can keep just one of the speaker articles and generate the rest using the indexing from the `moreSpeaker` object you created previously. 🤓

**Review by rpire — COMMENTED** [review]
## Greetings @mmatongo 👋

I can see the effort you have put into your capstone project, it is well structured and everything is working the way it should, congratulations! 👏🎉

However, there is one part of the code that could be improved even further in terms of readability and efficiency. Kindly take a look at the comment below for more details. ♻

Once again, great job, and congratulations on approving the capstone project! 😎👍

Cheers and Happy Coding 🚀🚀

**Pull request merged**